analyses/2019_04_CoderT_Adblocker and Tracker Blocker Analysis

```diff
@@ -0,0 +1,35 @@
+What is the difference between ad blocker dataset and tracker blocker dataset.#86
+
+I found a very interesting article https://www.sba-research.org/wp-content/uploads/publications/block_me_if_you_can.pdf 
+
+Summary of Article
+•	This looks at the links between ad blocking and tracker blocking on the basis of overall privacy and security of Internet users and which tools are the most effective.
+•	It looks at the tools that internet users use to block adds and to block tracking.
+•	come from the browser extensions which differentiate between tracking and non-tracking HTTP requests.  This will block the browser extensions and allow the non-tracking HTTP requests. 
+
+Blocking Trackers
+We know that network-based blocking is done through DNS blocking where addresses are blacklisted in order to block access to certain domains.  This will include ads and this method works independently of the used application.  The DNS filtering can be used to block entire domains but not individual URIs.
+
+When analyzing the data set, in relation to #24 – How does ad blocking relate to this dataset, we are sure to find entire domains being blocked because of the use of DNS blocking, and an analysis of how to factor this into the code to build a heuristic of browser attributes should be considered.  
+
+With Blocking Trackers and Interception Proxies, third party content cannot always be reliably detected at a network level, which might skew the results when looking at ad blocking in a dataset.  
```

Comment:
I see the bit of the paper you're summarizing. I'm very happy to see you summarizing in your own words and keeping things short - short is good! This may be a little too short though. I needed to go and read the paper to understand what you were referring to.